sets.json

```diff
@@ -232,7 +232,7 @@
   },
   {
     "id": "10",
-    "name": "Reflections II",
```

Comment:
Is there no Reflections I / Reflections 1?

Reply:
I honestly don't remember.  Just weird we'd have II and III without a I.

Reply:
Reflections I has no unique cards, only foil versions of cards from other sets.  If we ever expanded the "Printings" field to include things like foils, AIs, then it would make sense to create a Reflections I entry, so the foil versions of cards could refer to it.